### PR TITLE
feat: Implemented Continuous Integration(CI) pipeline for automated/manual build workflow

### PR DESCRIPTION
**Pull request by utkarsh006**
- Solves Issue #134

### CODE REVIEW

**Comment by utkarsh006** [comment]
@Kaaveh Kindly Review.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+name: CI
```
Comment (Kaaveh) [inline .github/workflows/ci.yaml:1]: There is a file for CI named `build.yml`, why you add new file for CI?

**Review by Kaaveh — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (utkarsh006) [reply .github/workflows/ci.yaml:1]: @Kaaveh I just used it for the convention

**Comment by utkarsh006** [comment]
@Kaaveh Hey what's the status? Does everything look fine?

**Comment by Kaaveh** [comment]
> @Kaaveh Hey what's the status? Does everything look fine?

Hi,
@alibahaaa will review this PR soon. Sorry for delay. 🙌🏻

**Comment by utkarsh006** [comment]
@Kaaveh Any updates?

**Comment by Kaaveh** [comment]
Sorry, it took so long @utkarsh006 , right now we are a little busy managing some tasks. I'll try to review it today.
Thanks for your patience.

**Review by Kaaveh — APPROVED** [review]

**Comment by Kaaveh** [comment]
@utkarsh006 Thanks for your contribute! 🙌🏻 We add more issues soon. Please check it out if you like.

**Comment by utkarsh006** [comment]
@Kaaveh Sure ! Would love to solve more from issues :)

**Comment by utkarsh006** [comment]
@Kaaveh Do give this PR a label as `hacktoberfest-accepted`
Check the guidelines : https://hacktoberfest.com/participation/#maintainers

**Comment by Kaaveh** [comment]
@utkarsh006 Done! 🙌🏻

**Pull request merged**